doc/data/messages/d/deprecated-module/bad.py

```diff
@@ -0,0 +1,16 @@
+from copy import deepcopy  # [deprecated-module]
```

Comment:
``deepcopy`` is not the best example as there's no drop in replacement that will make the code in good an equivalent. Better use ``os.path`` => ``Path`` or ``distutils`` => ``setuptools`` or ``optparse`` => ``argparse``.

Reply:
I understand, I just chose that tool because I noticed the deprecated-module setting in the pylintrc file which made me realize how to cause the error to happen.

Reply:
So, to confirm before committing, I should change bad.py to "import distutils # [deprecated-module]", and good.py to "import setuptools" and pylintrc to "[main]
py-version=3.7" to make the example better?

Reply:
Ok, I am using distutils and setuptools